src/provider/cloud-element-templates/properties/CustomProperties.js

```diff
@@ -2,16 +2,15 @@ import {
   find,
   forEach,
   groupBy,
-  isString,
-  isUndefined
+  isString
 } from 'min-dash';
 
-import { getBusinessObject } from 'bpmn-js/lib/util/ModelUtil';
-
 import { useService } from '../../../hooks';
 
 import { PropertyDescription } from '../../element-templates/components/PropertyDescription';
 
+import { getPropertyValue, setPropertyValue } from '../util/propertyUtil';
```

Comment:
Is this cloud element template specific? If so, let's make it a dedicated cloud element templates util. Otherwise it is OK to have it in the shared util space.

Reply:
It is a cloud element template specific and it's in the `cloud-element-templates/util` folder

Reply:
Thanks!